src/plugins/data/server/index_patterns/index.ts

```diff
@@ -20,4 +20,5 @@ import * as indexPatterns from './utils';
 
 export { IndexPatternsFetcher, FieldDescriptor, shouldReadFieldFromDocValues } from './fetcher';
 export { IndexPatternsService } from './index_patterns_service';
+export { getFieldsForWildcard } from './lib/fields_for_wildcard';
```

Comment:
I find it quite confusing - 
The whole point of the `indexPatterns` namespace was to be a single place to have all of the utility functions.
But it seems like in this PR you're exporting `getFieldsForWildcard` in parallel to having `indexPatterns.getFieldsForWildcard`?

Let me know if I understand correctly

Reply:
Thanks for pointing this out @lizozom! Two things:

1 - Bad naming on my part. They are not the same 😄 
2 - That being said, I missed that the `IndexPatternsFetcher` class is already being exported. I think I should be able to use that for what I need. 

I'm going to close this PR.